capa/features/extractors/dnfile/insn.py

```diff
@@ -254,7 +228,6 @@ def extract_features(fh: FunctionHandle, bbh: BBHandle, ih: InsnHandle) -> Itera
     extract_insn_property_features,
     extract_insn_number_features,
     extract_insn_string_features,
-    extract_insn_namespace_features,
-    extract_insn_class_features,
```

Comment:
combine into 1 function making it easier to maintain the code (since we're looking at the same set of CIL instructions).